modules/pubmaticBidAdapter.js

```diff
@@ -381,7 +389,7 @@ export const spec = {
               height: bid.h,
               creativeId: bid.crid || bid.id,
               dealId: bid.dealid,
-              currency: CURRENCY,
+              currency: DEFAULT_CURRENCY,
```

Comment:
The adapter is always returning bids in USD even though when bidfloorcurrency is set, bids can be returned in other currencies. The bid currency should be response.body.cur